### PR TITLE
OS LWC Changes

### PR DESCRIPTION
**Pull request by jfgarcia268**
OS LWC Activation Changes:
- Timeout values
- Change page.waitFor() (Deprecated) to await page.waitForTimeout()
- Log Messages
- Handle particular error when LWC OS Compile failed because of missing custom LWC
- Update jobInfo when LWC Activation fails so the error will show at the end
- Check if Manage Package is Supported

### CODE REVIEW

File: lib/datapacktypes/omniscript.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var vlocity = require('../vlocity');
 var puppeteer = require('puppeteer-core');
 var fs = require('fs-extra');
 var path = require('path');
+const chromium = require('chromium');
```
Comment (rutlabaga) [inline lib/datapacktypes/omniscript.js:5]: We won't bundle chromium as it will increase the size of the app too much.

Comment (jfgarcia268) [reply lib/datapacktypes/omniscript.js]: Done.. I have removed chromium and Other small changes @rutlabaga

File: lib/datapacktypes/omniscript.js [hunk 2]
```diff
@@ -90,14 +91,28 @@ OmniScript.prototype.afterActivationSuccess = async function(inputMap) {
     var jobInfo = inputMap.jobInfo;
     var dataPack = inputMap.dataPack;
 
+    if(!jobInfo.checkLWCActivation){
```
Comment (rutlabaga) [inline lib/datapacktypes/omniscript.js:94]: Should only check `if (!jobInfo.ignoreLWCActivation && !jobInfo.checkLWCActivation)`

File: lib/datapacktypes/omniscript.js [hunk 3]
```diff
@@ -90,14 +91,28 @@ OmniScript.prototype.afterActivationSuccess = async function(inputMap) {
...
+        var minVersions = yaml.safeLoad(fs.readFileSync(path.join(__dirname,'..', 'buildToolsMinVersionOS-LWC.yaml'), 'utf8'));
```
Comment (rutlabaga) [inline lib/datapacktypes/omniscript.js:95]: Should move this to a static variable or to another file so it only loads one.

**Pull request merged**